src/components/TodoList/TodoList.tsx

```diff
@@ -1 +1,17 @@
-export const TodoList = () => {};
+import { FC } from 'react';
+import { TodoInfo } from '../TodoInfo';
+import { ITodoInfo } from '../../types/ITodoInfo';
+
+type Props = {
+  todos: ITodoInfo[];
+};
+
+export const TodoList: FC<Props> = ({ todos }) => {
+  return (
+    <section className="TodoList">
+      {todos.map(todo => {
+        return <TodoInfo key={todo.id} todo={todo} />;
```

Comment:
```suggestion
      {todos.map(todo => (
        <TodoInfo key={todo.id} todo={todo} />
```
Return is not necessary here